src/documentation/ntfs2gtfs.md

```diff
@@ -0,0 +1,147 @@
+# NTFS to GTFS conversion
+## Introduction
+This document describes how a [NTFS feed](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/) is transformed into a [GTFS feed ](https://developers.google.com/transit/gtfs/reference/) in Navitia Transit Model.
+
+The resulting GTFS feed is composed of the following objects:
+- [agency](#agency.txt)
+- [routes](#routes.txt)
+- [stops](#stops.txt)
+- [trips](#trips.txt)
+- [stop_times](#stop_times.txt)
+- [calendar_dates](#calendar_dates.txt)
+- [transfers](#transfers.txt)
+- [shapes](#shapes)
+- [stop_extensions](#stop_extensions.txt): additional information providing the complementary stop codes used in external systems.
+
+## Mapping between NTFS and GTFS objects
+#### agency.txt
+
+GTFS field | Required | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+agency_id | yes | networks.txt | network_id | 
+agency_name | yes | networks.txt | network_name | 
+agency_url | yes | networks.txt | network_url | 
+agency_timezone | yes | networks.txt | network_timezone | `Europe/Paris` if the value is not provided.
+agency_lang | no | networks.txt | network_lang | 
+agency_phone | no | networks.txt | network_phone | 
+
+#### routes.txt
+
+GTFS field | Required | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+route_id | yes | lines.txt | line_id | 
+agency_id | no | lines.txt | network_id | (link to the [agency.txt](#agency.txt) file)
+route_short_name | yes | | lines.txt | line_code | 
+route_long_name | yes | lines.txt | line_name | 
+route_type | yes | | | The corresponding physical mode of the trips of the line, otherwise the default value is `3`. See the table below for the mapping of modes.
+route_color | no | lines.txt | line_color | 
+route_text_color | no | lines.txt | line_text_color | 
+route_sort_order | no | lines.txt | line_sort_order | 
+
+Each line of this file corresponds to a transit line modeled in the GTFS feed. In case a transit line uses more than one modes of transportation, it should be modeled separately for each different mode, according to the mapping of modes presented below. The priorities follow the [NeTex Specification](http://www.normes-donnees-tc.org/wp-content/uploads/2014/05/NF_Profil_NeTEx_pour_les_arrets-_F-_-_v2.pdf) (cf. chapter 6.2.3). 
```

Comment:
I think it would be better to move this part before the mapping description as the understanding of the mapping depens on it.
`Each line of this file corresponds to a transit line modeled in the GTFS feed.` : is it `NTFS feed` ?